replace_app_name.py

```
@@ -0,0 +1,98 @@
+        # Update files inside the renamed directory
+        for py_file in new_dir.rglob("*.py"):
+            update_file(py_file, old_name, new_name)
+    
+    # Update other files
+    for file_path in files_to_update:
+        full_path = project_root / file_path
+        if full_path.exists():
+            update_file(full_path, old_name, new_name)
+    
+    print(f"✅ Successfully replaced '{old_name}' with '{new_name}'")
+    print(f"📝 Don't forget to update your INSTALLED_APPS and URL imports!")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)